Tests/PointerStar.Shared.Tests/PointerStar.Shared.Tests.csproj

```diff
@@ -4,6 +4,10 @@
     <TargetFramework>net7.0</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Majorsoft.Blazor.Components.Modal" Version="1.5.0" />
```

Comment:
Do we need this in the shared project? I would only expect it to be needed in the Client project